File: Dicoding/Undefined/MyTestingApp/app/src/main/java/com/dicoding/mytestingapp/MainActivity.java
package com.dicoding.mytestingapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

import com.bumptech.glide.Glide;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{
    private Button btnSetValue;
    private TextView tvText;
    private ArrayList<String> names;
    private ImageView imgPreview;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        tvText = findViewById(R.id.tv_text);
        btnSetValue = findViewById(R.id.btn_set_value);
        btnSetValue.setOnClickListener(this);

        names = new ArrayList<>();
        names.add("Hendri");
        names.add("Kevin");
        names.add("Darto");

        imgPreview = findViewById(R.id.img_preview);
        //imgPreview.setImageDrawable(ContextCompat.getDrawable(this, R.drawable.fronalpstock_big));
        Glide.with(this).load(R.drawable.fronalpstock_big).into(imgPreview);
    }

    @Override
    public void onClick(View v) {
        if (v.getId() == R.id.btn_set_value){
            //tvText.setText("19");
            StringBuilder name = new StringBuilder();
            for (int i=0; i < names.size(); i++){
                name.append(names.get(i)).append("\n");
            }
            tvText.setText(name.toString());
        }
    }

}

File: Dicoding/Jetpack Pro/MyIdlingResource/app/src/main/java/com/hendri/myidlingresource/MainActivity.kt
package com.hendri.myidlingresource

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button.setOnClickListener {
            delay1()
        }
    }

    private fun delay1() {
        Handler().postDelayed({
            text_view.text = getString(R.string.delay1)
        }, 2000)
    }
}